### ISSUE TITLE
PATRONI_LOG_* environment variables are ignored.

### ISSUE THREAD

**Issue by mhaley-tignis** [zalando/spilo#718]
It seems that most environment variables are removed before starting the patroni process. I was attempting to configure patroni to log to a directory (instead of stdout) by using the `PATRONI_LOG_DIR` environment variable, but the `runit/patroni/run` script seems to be working against me here:

```
# Only small subset of environment variables is allowed. We don't want accidentally disclose sensitive information
for E in $(printenv -0 | tr '\n' ' ' | sed 's/\x00/\n/g' | grep -vE '^(KUBERNETES_(SERVICE|PORT|ROLE)[_=]|((POD_(IP|NAMESPACE))|HOSTNAME|PATH|PGHOME|LC_ALL|ENABLE_PG_MON)=)' | sed 's/=.*//g'); do
    unset $E
done
```
ref: https://github.com/zalando/spilo/blob/master/postgres-appliance/runit/patroni/run#L24-L26

Would it be possible to add `|(PATRONI_LOG_.*=)` to the regex to allow PATRONI_LOG_* variables to be passed to the process?

**Comment by CyberDem0n** [comment]
You can always inject part of Patroni configuration via `SPILO_CONFIGURATION`:
```json
{"log":{"level":"DEBUG", ...}}
```

**Comment by CyberDem0n** [comment]
> Is there a reason we don't want to allow PATRONI_* variables to be passed to patroni?

Yes, these variables may have some sensitive data that will be visible to Postgres. Exactly this is written in the comment to lines you are referring at: https://github.com/zalando/spilo/blob/823f1462219a8c0b28a63170987dd7ccdfedc3e0/postgres-appliance/runit/patroni/run#L23

**Comment by mhaley-tignis** [comment]
Made a quick PR here (https://github.com/zalando/spilo/pull/719) @CyberDem0n, let me know what you think.

**Comment by mhaley-tignis** [comment]
@CyberDem0n any additional thoughts here? Are we unable to rely on patroni process to protect itself?
If not, an alternative solution is here https://github.com/zalando/postgres-operator/pull/1857. Either one solves my needs.

**Comment by holooloo** [comment]
any news to decrease log verbosity?
